patch_rebaser/patch_rebaser.py

```diff
@@ -1,8 +1,11 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """Main module."""
 
-import configparser
+try:
+    import configparser
+except ImportError:  # Python 2
+    import ConfigParser as configparser
```

Comment:
Thanks you for doing it this way instead of using something like six.

Reply:
Seemed overkill for such small changes indeed!